#Debug Message
#GPS_Data
float32 curr_lat
float32 curr_long
int64 elapsed_time
int16 sats
float32 speed_val
float32 heading
#Diagnostics data
int16 bin_fullness
int16 battery
float32 l_motor
float32 r_motor
#Next waypoint
float32 dest_lat
float32 dest_long
int16 waypoint_id
#Heading and Destination
float32 head_to_dest
float32 dist_to_dest
float32 head_error
int16 current_state